Makefile

```diff
@@ -4,7 +4,7 @@ CARGO = CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse RUSTFLAGS="--cfg tokio_unstab
 all: build check-fmt check-clippy test
 
 .PHONY: test
-test:
+test: update
```

Comment:
I don't think we want to run update as part of these tasks. Adding an update step is fine, but we should run `make update` and commit the lockfile in a commit. I suspect this will cause unstaged changes in CI that could fail the pipeline.